src/containers/Settings/WebPaymentsSupport.js

```diff
@@ -0,0 +1,50 @@
+import React, {Component} from 'react'
+
+export default class WebPaymentsSupport extends Component {
+  constructor(props) {
+    super(props)
+    this.supported = (typeof PaymentRequest !== 'undefined')
+    this.handleClick = this.handleClick.bind(this)
```

Comment:
We usually use arrow functions instead of binding here, so `handleClick() {` would look like `handleClick = () => {`.